admin-dev/themes/new-theme/js/app/pages/stock/router/index.js

```diff
@@ -33,7 +33,7 @@ export default new VueRouter({
   mode: 'history',
   base: (() => {
     const hasIndex = /(index\.php)/.exec(window.location.href);
-    return `${window.data.baseUrl}${hasIndex ? '/index.php' : ''}/stock`;
+    return `${window.data.baseUrl}${hasIndex ? '/index.php' : ''}/sell/stocks`;
```

Comment:
note that I'm not proud of my fix neither to the current code, but fixing it "the right way" should be done in a specific pull request.